pkg/prometheus/query_range.go

```diff
@@ -0,0 +1,149 @@
+// Copyright 2020 IBM Corp.
```

Comment:
It looks like you sort of moved the original query.go but didn't. You moved the client.go file correctly. We'd want to collapse query_range.go and query.go into one file and not split the code up where one change won't replicate into the other.

Reply:
Got it, yea meant to get back to this after the preview query stuff was done.